source/src/enet_encap/cpf.c

```diff
@@ -271,7 +271,7 @@ EipStatus CreateCommonPacketFormatStructure(const EipUint8 *data,
       return kEipStatusError;
     }
 
-    CipUsint address_item_count = common_packet_format_data->item_count - 2;
+    unsigned int address_item_count = common_packet_format_data->item_count - 2U;
```

Comment:
why do you want to increase the width? the specification states its a USINT

Reply:
Oh sorry, you are right. its an UINT.
Could we please use CipUint to create a direct match with the specification?
It helpes me a lot to keep track which variables are part of the interface, and which are just part of the implementation

Reply:
Try 417202a.